src/systems/a320_systems/src/flaps_computer.rs

```diff
@@ -0,0 +1,1347 @@
+use systems::simulation::{
+    Read, SimulationElement, SimulationElementVisitor, SimulatorReader, SimulatorWriter,
+    UpdateContext, Write,
+};
+
+use uom::si::{angle::degree, f64::*, velocity::knot};
+
+//The different flaps configurations
+#[derive(Debug, Copy, Clone, PartialEq)]
+enum FlapsConf {
+    Conf0 = 0,
+    Conf1 = 1,
+    Conf1F = 2,
+    Conf2 = 3,
+    Conf3 = 4,
+    ConfFull = 5,
+}
+
+impl From<usize> for FlapsConf {
+    fn from(value: usize) -> Self {
+        match value {
+            0 => FlapsConf::Conf0,
+            1 => FlapsConf::Conf1,
+            2 => FlapsConf::Conf1F,
+            3 => FlapsConf::Conf2,
+            4 => FlapsConf::Conf3,
+            _ => FlapsConf::ConfFull,
```

Comment:
**suggestion**: Any number other than 0 to 4 will match to be `ConfFull`. In my experience failing early rather than hiding problems is usually a good idea, thus I'd suggest matching on `5` and having an `_` match which panics.

Reply:
Note the typo in:
```rust
_ => panic!("Invalid configuation"),
```

I'd also suggest making the message more specific, to make debugging easier. E.g.: `"Cannot convert from {} to FlapsConf."`.

Reply:
Suggestion implemented in `5b14499 `